src/Workspaces/CSharp/Portable/CodeGeneration/ParameterGenerator.cs

```diff
@@ -13,7 +13,7 @@
 using Microsoft.CodeAnalysis.Shared.Extensions;
 
 using Microsoft.CodeAnalysis.PooledObjects;
-using static Microsoft.CodeAnalysis.CodeGeneration.CodeGenerationHelpers;
+using static Microsoft.CodeAnalysis.CSharp.CodeGeneration.CSharpCodeGenerationHelpers;
```

Comment:
is this the namespace used by the sibling types?

Reply:
It's used by a static method